### ISSUE TITLE
Maxima keeps crashing on opening

### ISSUE THREAD

**Issue by piscessafespace** [wxMaxima-developers/wxmaxima#1738]
After usually 2 minutes of using wxmaxima it crashes down, the window just closes, no error message, nothing. My school requires me to use wxMaxima but since a Windows Update it doesnt work anymore. What can I do?

**Comment by gunterkoenigsmann** [comment]
Do you have an gdb64 installed? And what is the exact message or symptom you get?

**Comment by gunterkoenigsmann** [comment]
Hmm... ...I've just seen that you wrote that it just closes without outputting any message.

Does the wxMaxima Nightly Build work instead for you? An installer for it can be found at https://wolfgang.dautermann.at/maxima/nightlybuild/

**Comment by piscessafespace** [comment]
> Hmm... ...I've just seen that you wrote that it just closes without outputting any message.
> 
> Does the wxMaxima Nightly Build work instead for you? An installer for it can be found at https://wolfgang.dautermann.at/maxima/nightlybuild/

Which file am I supposed to download? just: "[maxima-current-win64.exe](https://wolfgang.dautermann.at/maxima/nightlybuild/maxima-current-win64.exe)" ?

**Comment by richardgobeli** [comment]
Yes, that is the one I always use.  First, delete your last version.  It will get installed in the folder "C:\maxima-current\" automatically.
The start file is in the bin folder. WxMaxima.exe.

**Comment by piscessafespace** [comment]
Can someone please explain to me why this version of Maxima doesnt crash anymore? I am really curious because I have tried almost everything until now and nothing seemed to work. Thanks so much for all the help. Please take this [cute cat picture](https://user-images.githubusercontent.com/97235471/209848094-d9fe6188-8f23-4a45-aa0d-d4a7ac507549.png) as an offering.

Have a beautiful and restful night to everyone

**Comment by gunterkoenigsmann** [comment]
Wolfgang fixed many bugs in unicode handling in log messages. Maybe that was the issue. But as the program if I understand it right just closed silently perhaps the problem was that we used two methods to assign IDs to events - and that the close event somehow collided with some event that would eventually happen. I still have no idea how we caused a event collision. But it seems that using only one method for assigning event IDs did resolve several issues that only happened on some computers while they weren't reproducible on others. Another potential candidate was a race condition with the background jobs: I resolved some of them, too.
